pages/about.html

```diff
@@ -4,9 +4,73 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../stylesheets/style.css" rel="stylesheet">
+  <link href="../stylesheets/about.css" rel="stylesheet">
+  <title>About Me</title>
 </head>
 <body>
-  
+  <header>
+    <h1>
+      About Me
+    </h1>
+    <nav>
+      <a href="./index.html">Home</a>
+      <a href="./portfolio.html">My Work</a>
+    </nav>
+  </header>
+  <main>
+    <h2>Programming</h2>
+    <p>Computer games and logic puzzles drove my interest in programming 
+      as a teenager. As an adult I also appreciate 
+      the problem solving elements and thinking critically with an eye towards 
+      efficient design.
+    </p>
```

Comment:
"Programming", "My Other Interests" and "My Pets" all feel like distinct sections and have heading tags. You could wrap them in section tags which would bring some more semantic meaning to the HTML here. 

https://www.thoughtco.com/html5-section-element-3467994 if you feel like reading more